ecs-agent/api/resource/ebs_discovery_linux.go

```diff
@@ -0,0 +1,99 @@
+//go:build linux
+// +build linux
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package resource
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// LsblkOutput is used to manage and track the output of `lsblk`
+type LsblkOutput struct {
+	BlockDevies []BD `json:"blockdevices"`
+}
+type BD struct {
```

Comment:
nit non blocking: If this type is used outside of this file I would prefer to name it `BlockDevice`. It seems not to be the case though (as far as I understood)

Reply:
Yep, that's correct.

Reply:
agree. Can we change the name? -- it's ok to refer to this as `bd` in method receivers.

Reply:
Ack, will update the name to `BlockDevice` instead. Thanks!